readme.md

```diff
@@ -276,6 +276,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 ## Related Projects
 Other awesome open source projects that use Lighthouse.
 
+* **[lighthouse-gh-reporter](https://github.com/addyosmani/webpack-lighthouse-plugin)** - give Lighthouse-CI the power of put reports as a comment on your pull requests.
```

Comment:
Checking the history of this file, we add new integrations to the bottom of the list usually.  So I'd move this to right under lighthouse4u.

Reply:
nit: give Lighthouse-CI the power ~~of put~~ _to make_ reports as a comment on your pull requests _in Github_.